lib/datadog/tracing/contrib/grape/endpoint.rb

```diff
@@ -62,6 +63,15 @@ def endpoint_start_process(_name, _start, _finish, _id, payload)
               span.set_tag(Tracing::Metadata::Ext::TAG_COMPONENT, Ext::TAG_COMPONENT)
               span.set_tag(Tracing::Metadata::Ext::TAG_OPERATION, Ext::TAG_OPERATION_ENDPOINT_RUN)
 
+              if (grape_route = env['grape.routing_args'] && env['grape.routing_args'][:route_info])
```

Comment:
Can you add parenthesis here to improve reading clarity in this `if` condition?
```suggestion
              if ((grape_route = env['grape.routing_args']) && grape_route[:route_info])
```